pkg/limatype/lima_yaml.go

```diff
@@ -52,11 +52,12 @@ type LimaYAML struct {
 	PropagateProxyEnv *bool          `yaml:"propagateProxyEnv,omitempty" json:"propagateProxyEnv,omitempty" jsonschema:"nullable"`
 	CACertificates    CACertificates `yaml:"caCerts,omitempty" json:"caCerts,omitempty"`
 	// Deprecated: Use VMOpts.VZ.Rosetta instead.
-	Rosetta              Rosetta `yaml:"rosetta,omitempty" json:"rosetta,omitempty"`
-	Plain                *bool   `yaml:"plain,omitempty" json:"plain,omitempty" jsonschema:"nullable"`
-	TimeZone             *string `yaml:"timezone,omitempty" json:"timezone,omitempty" jsonschema:"nullable"`
-	NestedVirtualization *bool   `yaml:"nestedVirtualization,omitempty" json:"nestedVirtualization,omitempty" jsonschema:"nullable"`
-	User                 User    `yaml:"user,omitempty" json:"user,omitempty"`
+	Rosetta              Rosetta        `yaml:"rosetta,omitempty" json:"rosetta,omitempty"`
+	Plain                *bool          `yaml:"plain,omitempty" json:"plain,omitempty" jsonschema:"nullable"`
+	TimeZone             *string        `yaml:"timezone,omitempty" json:"timezone,omitempty" jsonschema:"nullable"`
+	NestedVirtualization *bool          `yaml:"nestedVirtualization,omitempty" json:"nestedVirtualization,omitempty" jsonschema:"nullable"`
+	User                 User           `yaml:"user,omitempty" json:"user,omitempty"`
+	Meta                 map[string]any `yaml:"meta,omitempty" json:"meta,omitempty" jsonschema:"nullable"`
```

Comment:
`FillDefault` needs to be updated for overriding

Reply:
> Bikeshedding: why not call this "labels"

1. Because I think of the value of labels as just strings (`map[string]string`), not objects, and I don't want to put any constraints on the data a user wants to put there.

2. I find `meta` more generic than `labels`, e.g. `meta.keywords` looks better to me than `labels.keywords`. And with point (1), `meta.keywords` can be a `[]string`, but `labels.keywords` would be a comma-separated list that would need to be parsed and could not be used easily with `jq`/`yq`.

Reply:
> `FillDefault` needs to be updated for overriding

I guess. And then it also should be supported by template embedding, for consistency.

At first I thought since Lima doesn't use the value of `Meta`, just declaring the field to make sure we don't get strict YAML errors would be enough. But I guess I was wrong. 😄